e2e/src/config/tests-config/environments/local.ts

```diff
@@ -33,7 +33,9 @@ const config: Config = {
     besuNodeRpcUrl: L2_BESU_NODE_RPC_URL,
     chainId: 1337,
     l2MessageServiceAddress: "0xe537D669CA013d86EBeF1D64e40fC74CADC91987",
+    // Nonce 10
```

Comment:
What does this comment mean?

Reply:
So in this file we have multiple hardcoded contract deployment addresses
- I was stuck for a little while on where these hardcoded contract deployment addresses came from.
- It turns out that the contract deployment address is a function of i.) The deployer address, and ii.) The deployer nonce
- These comments are referring to the nonce used to compute the hardcoded contract address